pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: rh-push-to-registry-redhat-io
   labels:
-    app.kubernetes.io/version: "1.1.1"
+    app.kubernetes.io/version: "1.1.2"
```

Comment:
I think it should be `1.2.0` :slightly_smiling_face:

Reply:
fixed!